global.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
@adityapatwardhan, your last commit had 2 failures in `PowerShell-CI-windows`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @{InputObject=System.Security.Principal.dll; SideIndicator=<=}.
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\Startup.Tests.ps1: line 120
120:         $diffs | Should -BeExactly $null
```
Get-Member.Should be able to be called on IntPtr
```powershell
Expected exactly 2, but got 4.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Get-Member.Tests.ps1: line 46
46:         $results.Count | Should -BeExactly 2
```